test/blueprints/Do.Test.Blueprints.Service.Test/DataAccess/NHibernateUserType/CorruptedData.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Do.Test.DataAccess.NHibernateUserType;
+
+public class CorruptedData : TestServiceSpec
+{
+    [Test]
+    public void Special_characters_do_not_cause_corrupted_data_for_object_user_type_properties()
```

Comment:
when naming prefer what it allows, not what it does not cause, e.g., 

`Object_user_type_allows_special_characters_to_be_used_within_strings`